### ISSUE TITLE
[Accessibility]: Batches close page: Most icons have no label

### ISSUE THREAD

**Issue by eseliger** [sourcegraph/sourcegraph#35012]
### Audit type

ARC Toolkit

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34089

### Problem description

<img width="1137" alt="Screenshot 2022-05-05 at 21 20 35@2x" src="https://user-images.githubusercontent.com/19534377/167009933-7bbfebd0-836f-401e-b851-9efb85d6b12a.png">

### Expected behavior

They do have a label and ARC is happy.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
Fixed through work on https://github.com/sourcegraph/sourcegraph/issues/34582
